includes/gateways/stripe/includes/payment-methods/class-give-stripe-checkout.php

```diff
@@ -306,11 +306,12 @@ public function process_checkout( $donation_id, $data ) {
 			give_update_meta( $donation_id, '_give_stripe_donation_summary', $donation_summary );
 
 			// Redirect to show loading area to trigger redirectToCheckout client side.
-			wp_safe_redirect(
+			wp_redirect(
```

Comment:
Any reason we wouldn't continue using `wp_safe_redirect` here?

Reply:
Reverted the change as not an intended change